variants/STM32F0xx/F042C(4-6)(T-U)/PeripheralPins.c

```diff
@@ -199,15 +199,15 @@ WEAK const PinMap PinMap_SPI_SSEL[] = {
 
 //*** CAN ***
 
-#ifdef HAL_CAN_MODULE_ENABLED
+#if defined(HAL_CAN_MODULE_ENABLED) || defined(HAL_CAN_LEGACY_MODULE_ENABLED)
```

Comment:
All generic PeripheralPins.c have been changed. Great.
Is it necessary to apply the same change in custom PeripheralPins.c like for example PeripheralPins_BLACK_F407VX.c ?